pos_order_printer_product/doc/index.rst

```diff
@@ -0,0 +1,27 @@
+===========================
+ POS Order Printer Product
+===========================
+
+Installation
+============
+
+* `Install <https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html>`__ this module in a usual way
+
+Configuration
+=============
+
+* Go to ``Point of Sale >> Configuration >> Order Printers``
+
+  * Click on ``[Create]``
+  * Specify a name of new printer in the "Printer Name" field
+  * Specify the IP address of printer
+  * Select Product Categories
+  * Select Products
+  * Click on ``[Save]``
```

Comment:
Use formatting when reference to fields: https://odoo-development.readthedocs.io/en/latest/dev/docs/rst-requirements.html